enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/provider/EmptyOnboardingProvider.java

```diff
@@ -27,7 +27,7 @@
 class EmptyOnboardingProvider implements OnboardingProvider {
 
     @Override
-    public String lookupUser(LookupUserRequest request) throws OnboardingProviderException {
+    public LookupUserResponse lookupUser(LookupUserRequest request) throws OnboardingProviderException {
```

Comment:
This should be enough so far. We are able to receive the flag but do not propagate to the client yet.